data/person

```diff
@@ -84,3 +84,4 @@ Carter
 Acrogenous
 Whangd00dle
 Will BL
+Tehc
```

Comment:
Is this really necessary? I think this PR would be great without this one line

Reply:
get out.